### ISSUE TITLE
Custom tokens docs start off with irrelevant info

### ISSUE THREAD

**Issue by mitschabaude** [o1-labs/docs2#546]
Feedback from the community: It's not clear why we need both signature authorization and proof authorization for the token owner.

Indeed, signature auth for the token owner doesn't make sense in any application. And yet, the custom token docs page presents signature auth as the first of two possible ways to approve token interactions. This is confusing, the docs shouldn't waste user attention and brain space on something that is in theory supported by the protocol but doesn't have any use case. Instead, it should explain how to use protocol features in practice to solve real use cases.

Solution: Remove the section on signature authorization from the custom token docs. 

Also, the token docs don't explain how to do token transfers from a zkApp, i.e. how to use proof authorization for the _token user_.

In summary, here is the full matrix of different authorizations for different accounts:

* Signature for token owner + signature for token user: Irrelevant, currently covered in docs, FIX
* Signature for token owner + proof for token user: Irrelevant, not covered in docs, OK
* Proof for token owner + signature for token user: Important, covered in docs, OK
* Proof for token owner + proof for token user: Important, not covered in docs, FIX

**Comment by barriebyron** [comment]
@mitschabaude  you are the absolute best developer advocate and teammate! thank you for identifying these errors and gaps

**Comment by barriebyron** [comment]
PR #812
removes the Signature Proof section in the custom token doc 

Still outstanding:
Proof for token owner + proof for token user: Important, not covered in docs
